secedgar/filings/combo.py

```diff
@@ -0,0 +1,65 @@
+from secedgar.filings import MasterFilings, DailyFilings
+from datetime import date, timedelta
+from secedgar.utils import get_quarter, get_month
+import logging
+
+class ComboFilings:
```

Comment:
Is this supposed to solve the issue where somebody wants to get many days of filings but those days don't necessarily fit perfectly into a single quarter?

Reply:
If so, I think that this would be best handled with the wrapper around the different strategies we had discussed instead of making a new strategy (ComboFilingsStrategy is what this would be in the v0.4.0 version).

Reply:
Also, we are going to need to change imports if we change file from network_client to client, I believe.

Reply:
It’s essentially a function already, but we could make it so